Assets/LeapMotion/Scripts/Utils/Hands.cs

```diff
@@ -0,0 +1,168 @@
+    /// other two palm-basis directions, see RadialAxis and DistalAxis.
+    /// The direction out of the back of the hand would be called the dorsal axis. </summary>
+    public static Vector3 PalmarAxis(this Hand hand) {
+      return -hand.Basis.yBasis.ToVector3();
+    }
+
+    /// <summary> Returns the the direction towards the thumb
+    /// that is perpendicular to the palmar and distal axes.
+    /// Left and right hands will return opposing directions.
+    /// The direction away from the thumb would be called the ulnar axis. </summary>
+    public static Vector3 RadialAxis(this Hand hand) {
+      if (hand.IsRight) {
+        return hand.Basis.xBasis.ToVector3();
+      }
+      else {
+        return -hand.Basis.xBasis.ToVector3();
+      }
+    }
+
+    /// <summary> Returns the direction towards the fingers that is
+    /// perpendicular to the palmar and radial axes.
+    /// The direction towards the wrist would be called the proximal axis. </summary>
+    public static Vector3 DistalAxis (this Hand hand) {
+      return hand.Basis.zBasis.ToVector3();
+    }
+
+    /// <summary>
+    /// Returns whether the pinch strength for the hand is greater than 0.8.
+    /// For more reliable pinch behavior, consider applying hysteresis to the Hand.PinchStrength property.
+    /// </summary>
+    public static bool IsPinching(this Hand hand) {
+      return hand.PinchStrength > 0.8F;
+    }
+
+    /// <summary>
+    /// Returns the average of the index finger and thumb tip positions.
+    /// </summary>
+    public static Vector3 GetPinchPosition(this Hand hand) {
+      return (hand.Fingers[(int)Finger.FingerType.TYPE_INDEX].TipPosition + hand.Fingers[(int)Finger.FingerType.TYPE_THUMB].TipPosition).ToVector3() * 0.5F;
+    }
+
+    /// <summary>
+    /// Returns whether this vector faces from a given world position towards another world position within a maximum angle of error.
+    /// </summary>
+    public static bool IsFacing(this Vector3 facingVector, Vector3 fromWorldPosition, Vector3 towardsWorldPosition, float maxOffsetAngleAllowed) {
+      Vector3 actualVectorTowardsWorldPosition = (towardsWorldPosition - fromWorldPosition).normalized;
+      return Vector3.Angle(facingVector, actualVectorTowardsWorldPosition) <= maxOffsetAngleAllowed;
+    }
+
+    /// <summary>
+    /// Returns a confidence value from 0 to 1 indicating how strongly the Hand is making a fist.
+    /// </summary>
+    public static float FistStrength(this Hand hand) {
+      return (Vector3.Dot(hand.Fingers[1].Direction.ToVector3(), -hand.DistalAxis() )
+            + Vector3.Dot(hand.Fingers[2].Direction.ToVector3(), -hand.DistalAxis() )
+            + Vector3.Dot(hand.Fingers[3].Direction.ToVector3(), -hand.DistalAxis() )
+            + Vector3.Dot(hand.Fingers[4].Direction.ToVector3(), -hand.DistalAxis() )
+            + Vector3.Dot(hand.Fingers[0].Direction.ToVector3(),  hand.RadialAxis() )
```

Comment:
Shouldn't this be -hand.RadialAxis()?

Also why do they still have the parentheses?